pkg/infra/tower.go

```diff
@@ -122,3 +122,8 @@ func (t *Tower) ResetTower() {
 	t.currPlatFood = t.maxPlatFood
 	t.currPlatFloor = 1
 }
+
+func (t *Tower) GetCurrPlatFood() (float64, int) {
```

Comment:
- I think these should be two different functions for clarity?

Reply:
```suggestion
func (t *Tower) CurrPlatFood() (float64, int) {
```
In GoLang the convention for getters is straight the variable without the `Get`

Reply:
I've removed that function altogether as it was not needed (as per Jason's comment below), and I've changed the function name!